Readme.md

```diff
@@ -69,6 +69,12 @@ Installation is done using the
 $ npm install mysql
 ```
 
+You can also install mysqljs using the [Yarn package manager](https://yarnpkg.com/lang/en/docs/install/). Yarn is an alternative to npm, which enables a flat dependency tree.
```

Comment:
`mysqljs` is the name of the GitHub organization, not the module/product. Just call it `mysql` or "this module" here.